src/descriptor/tr.rs

```diff
@@ -0,0 +1,201 @@
+// Tapscript
+
+// use super::{
+//     checksum::{desc_checksum, verify_checksum},
+//     DescriptorTrait,
+// };
+use bitcoin::hashes::_export::_core::fmt::Formatter;
+use expression::{self, FromTree, Tree};
+use std::sync::Arc;
+use std::{fmt, str::FromStr};
+use Segwitv0;
+use {miniscript::Miniscript, Error, MiniscriptKey};
+
+// TODO: Update this to infer version from descriptor.
+const VER: u8 = 0xc0;
```

Comment:
In 73dd23e247dc47fd4d6d50244bc30e27abd3b487,
The above two line should be removed.